### ISSUE TITLE
Do you have plan to expand this projects' functions into stable diffusion domain ?

### ISSUE THREAD

**Issue by svjack** [sabetAI/BLoRA#2]
As the title say, expand it into diffuser project.

**Comment by sabetAI** [comment]
Yes, was originally planning to apply it to diffusion models first, but the peft library has some a more convenient api for injecting multiple LoRAs into the same model. Hoping to release the diffusers version in the next few weeks.
